requirments.txt

```diff
@@ -1,4 +1,7 @@
 requests
 pydantic
 tqdm
+pysbd
+spacy==0.2.1
```

Comment:
Woah. We don't use that old version of spacy. Use a stable version, something like v2?

Reply:
My brain clearly needs more coffee.